joomla/joomla-cms
# [4.1] Proper cache invalidation for css urls

Pull Request for Issue #32573 (partially) .

### Summary of Changes
- Adds one more PostCSS plugin https://www.npmjs.com/package/postcss-url-version
- Adjust the 2 scripts to use the above plugin


### Testing Instructions
Make a copy of the media and also the `css` folder of cassiopeia, atum and installtion
On this PR's code you need to run `npm install`
Then compare the produced css with the one you've backup before for example: search for `url(`, The new css files should have someting like `?v=13423-random-alphanumeric` at the end (only for local images/fonts)  


### Actual result BEFORE applying this Pull Request
Urls don;t have a hash


### Expected result AFTER applying this Pull Request
Urls have a hash

![Screenshot 2021-03-03 at 16 33 38](https://user-images.githubusercontent.com/3889375/109835013-18766f00-7c43-11eb-8a52-9a7af97cab98.png)



### Documentation Changes Required
No


One comment here, although at the original issue I mentioned another plugin there were bugs and also it's abandoned for some years. (I didn't either wanted to write a plugin, ok less than 100locs, but still)

@wilsonge is this ok for 4.0.x or should I rebase it to 4.0.x?
